### PR TITLE
LPS-189612 Remove feature flag LPS-163716 Default Value

### PR DESCRIPTION
**Pull request by murilostodolni**
Related: https://liferay.atlassian.net/browse/LPS-189612

### CODE REVIEW

**Comment by murilostodolni** [comment]
ci:stop

**Comment by murilostodolni** [comment]
ci:test:sf

**Comment by murilostodolni** [comment]
ci:test:relevant

**Comment by murilostodolni** [comment]
ci:test:object

File: modules/apps/object/object-api/src/main/java/com/liferay/object/exception/ObjectFieldDefaultValueException.java [hunk 1]
```diff
@@ -1,39 +0,0 @@
...
- */
-
-package com.liferay.object.exception;
-
-import com.liferay.portal.kernel.exception.PortalException;
-
-/**
- * @author Marco Leo
- */
-public class ObjectFieldDefaultValueException extends PortalException {
```
Comment (seltong) [inline modules/apps/object/object-api/src/main/java/com/liferay/object/exception/ObjectFieldDefaultValueException.java:22]: Can we really delete this exception? Could you check if this has been defined in the `service.xml`?

I believe this class will be created again if you run the command `gw buildService`. Could you please check it?

**Review by seltong — CHANGES_REQUESTED** [review]

File: modules/apps/object/object-api/src/main/java/com/liferay/object/exception/ObjectFieldDefaultValueException.java (hunk 1 above)
Comment (murilostodolni) [reply modules/apps/object/object-api/src/main/java/com/liferay/object/exception/ObjectFieldDefaultValueException.java:22]: We don't need this Exception anymore. The DefaultValue was remove as parameter of ObjectField. Now, the DefaultValue is a parameter of ObjectFieldSettings.

I will remove the `ObjectFieldDefaultValue` in the `service.xml`.

**Comment by murilostodolni** [comment]
ci:test:sf

**Review by seltong — APPROVED** [review]
LGTM

**Review by naraandrade — APPROVED** [review]

**Comment by murilostodolni** [comment]
ci:test:object

**Comment by murilostodolni** [comment]
ci:test:relevant

**Comment by murilostodolni** [comment]
ci:test:stable

**Comment by murilostodolni** [comment]
ci:forward:force

**Pull request closed without merging**